.github/workflows/test.yml

```diff
@@ -28,3 +28,12 @@ jobs:
           pip install .[dev]
       - name: Run unit tests
         run: PYTHONPATH=./src pytest
+      - name: Validate OTK configuration
+        run: otk compile example/fedora/minimal-40-x86_64.yaml
```

Comment:
Maybe use: `run: |` and do an otk validate for both the minimal-40-x86` and the `-aarch64` one? We *could* (but don't have to) use `otk -v validate ...` to get slightly more debug output if something fails.